### ISSUE TITLE
%install_ext deprecation

### ISSUE THREAD

**Issue by agentjwall** [cpcloud/ipython-autotime#3]
Upon installing and loading the extension with:

```
%install_ext https://raw.github.com/cpcloud/ipython-autotime/master/autotime.py
%load_ext autotime
```

The following error message is received:

```
UserWarning: %install_ext` is deprecated, please distribute your extension(s)as a python packages.
```

You might want to look into this to ensure it doesn't cause problems in the future. Thanks for making this though, it's awesome!

**Comment by Tagar** [comment]
+1 thanks for posting this.

**Comment by proinsias** [comment]
`%install_ext https://raw.github.com/cpcloud/ipython-autotime/master/autotime.py` doesn't seem to do anything in jupyter notebook 4.2.1.

**Comment by saeedesmaili** [comment]
+1

**Comment by timhughes** [comment]
In newer versions it gives
```UsageError: Line magic function `%install_ext` not found.```

**Comment by cpcloud** [comment]
Fixed in 23963fc
